src/main/java/dk/cngroup/lentils/service/PdfExportService.java

```diff
@@ -0,0 +1,92 @@
+package dk.cngroup.lentils.service;
+
+import com.itextpdf.text.BaseColor;
+import com.itextpdf.text.Document;
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.text.Element;
+import com.itextpdf.text.Font;
+import com.itextpdf.text.FontFactory;
+import com.itextpdf.text.Paragraph;
+import com.itextpdf.text.Phrase;
+import com.itextpdf.text.pdf.PdfPCell;
+import com.itextpdf.text.pdf.PdfPTable;
+import com.itextpdf.text.pdf.PdfWriter;
+import dk.cngroup.lentils.entity.view.TeamScore;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.util.List;
+import java.util.stream.Stream;
+
+@Service
+public class PdfExportService {
+    private static final int NUMBER_OF_TABLE_COLUMNS = 3;
+    private static final int PARAGRAPH_FONT_SIZE = 35;
+    private static final int SPACING_AFTER_PARAGRAPH = 20;
+
+    private final ScoreService scoreService;
+
+    @Autowired
+    public PdfExportService(final ScoreService scoreService) {
+        this.scoreService = scoreService;
+    }
+
+    public ByteArrayInputStream exportScoresToPdf() throws DocumentException {
```

Comment:
Tuto metodu bych dal private a vytvoril novou metodu, ktera bude public a pouze zavola `exportScoresToPdf` s tim, ze toto volani zabali to try/catch a v catch vyhodi novou vlastni unchecked exception, ktera se bude vracet az ke klientovi. Vyhoda toho je, ze pokud nekdy zmenime to, cim se generuje to PDF, tak muze nastat situace, ze ta nova technologie bude vyhazovat zase nejakou jinou vyjimku a tim padem by bylo dobre se s ni tady vyporadat a dal uz vracet jen tu nasi. Pokud to pak budem menit, zmenime jen tady tuto implementaci a nic jineho ;)